src/ray/common/ray_config_def.h

```diff
@@ -92,6 +92,12 @@ RAY_CONFIG(size_t, free_objects_batch_size, 100)
 
 RAY_CONFIG(bool, lineage_pinning_enabled, false)
 
+/// The maximum ratio of lineage to objects in scope to keep. If the amount of
```

Comment:
Can we instead bound by a hard cap like 1GB? Lineage is consuming heap memory, which isn't really related to objects / objet size.